packages/ra-ui-materialui/src/button/SortButton.tsx

```diff
@@ -114,7 +114,7 @@ const SortButton: FC<{ fields: string[]; label?: string }> = ({
                         data-sort={field}
                         key={field}
                     >
-                        {translate(field)}{' '}
+                        {translate(`resources.${resource}.fields.${field}`}{' '}
```

Comment:
```suggestion
                        {translate(`resources.${resource}.fields.${field}`)}{' '}
```

The translate function is not closed ;)

Reply:
I'm really sorry. Thank you for your patience.